### PR TITLE
fix(table): group action of children of table row

### LINKED ISSUE #3544: [BUG] - Disabled checkboxes in selectable table do not behaves as expected
### NextUI Version

2.4.6

### Describe the bug

When I use a `<Table>` with a non-undefined `selectionMode`, all the checkboxed with `isDisabled` or `isReadonly` switch state when clicking on a table row.

Here is athe code of my simple component:

```javascript
"use client";

import {
  Table,
  TableBody,
  TableCell,
  TableColumn,
  TableHeader,
  TableRow,
} from "@nextui-org/table";
import { Checkbox } from "@nextui-org/checkbox";

export default function Home() {
  return (
    <section>
      <h1>Selectable table: KO</h1>
      <Table selectionMode="single">
        <TableHeader>
          <TableColumn>Disabled</TableColumn>
          <TableColumn>Readonly</TableColumn>
        </TableHeader>
        <TableBody>
          <TableRow>
            <TableCell>
              <Checkbox isDisabled>Disabled</Checkbox>
            </TableCell>
            <TableCell>
              <Checkbox isReadOnly>Readonly</Checkbox>
            </TableCell>
          </TableRow>
        </TableBody>
      </Table>

      <h1>Unselectable table: OK</h1>
      <Table>
        <TableHeader>
          <TableColumn>Disabled</TableColumn>
          <TableColumn>Readonly</TableColumn>
        </TableHeader>
        <TableBody>
          <TableRow>
            <TableCell>
              <Checkbox isDisabled>Disabled</Checkbox>
            </TableCell>
            <TableCell>
              <Checkbox isReadOnly>Readonly</Checkbox>
            </TableCell>
          </TableRow>
        </TableBody>
      </Table>
    </section>
  );
}
```

### Your Example Website or App

https://github.com/nbusser/nextui-issue-checkbox-tables

### Steps to Reproduce the Bug or Issue

1. `pnpm i`
2. `npm run dev`
3. Go on `localhost:3000`

### Expected behavior

As a user, I expect being NOT able to change the state of a disabled/readonly checkbox within a table

### Screenshots or Videos

https://github.com/user-attachments/assets/334dde76-96d3-44f2-8208-d9a88edd4336

### Operating System Version

macOs

### Browser

Firefox

### PR DESCRIPTION
**Pull request by awesome-pro**
Closes #3544 

## 📝 Description

Fixed the group action of all the children of table row

## ⛳️ Current behavior (updates)

https://github.com/user-attachments/assets/f632eed2-ae71-403a-bae2-f05c661fb6f4

## 🚀 New behavior

https://github.com/user-attachments/assets/6b6227f6-68e7-4e86-a20a-d1f95fe3a6e0

## Summary by CodeRabbit

- **Bug Fixes**
  - Resolved navbar height issues for improved rendering and layout stability.
  - Enhanced the functionality of actions within table rows, ensuring expected behavior when interacting with child elements.

- **Style**
  - Adjusted class definitions for table headers and rows to improve visual hierarchy and accessibility.

### CODE REVIEW

**Comment by abhinandan-verma** [comment]
@wingkwong  sir review ?

File: .changeset/brown-days-applaud.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/theme": patch
+---
+
+fixed the navbar height when `h-full`(#1694)
```
Comment (wingkwong) [inline .changeset/brown-days-applaud.md:5]: how does it relate to this issue

**Comment by abhinandan-verma** [comment]
@wingkwong I am sorry, perhaps it was due small mistake

**Review by wingkwong — CHANGES_REQUESTED** [review]
By removing `group`, you will break all `group-*` styles. For example, `isStriped` styles will be removed because the parent is not defined.

**Review by TarasVasylivskyy — APPROVED** [review]

**Comment by wingkwong** [comment]
as discussed with Abhinandan, this pr will be closed and I'll take over it.

**Pull request closed without merging**